src/collection/collection.go

```diff
@@ -32,6 +32,7 @@ type SchemaList map[string]TableList
 type TableList map[string][]string
 
 type databaseIgnoreList map[string]struct{}
+type tableIgnoreList map[string]struct{}
 
```

Comment:
I would change this in favour of something generic like "ignoreList" to that we do not have `map[string]struct{}` in the signature

Reply:
fixed it in [65b6799](https://github.com/newrelic/nri-postgresql/pull/164/commits/65b67991c5b2a919033ae9cecdce5fe8c8709e73)